hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/handlers/ScheduleCreateHandler.java

```diff
@@ -164,8 +173,9 @@ public void handle(@NonNull final HandleContext context) throws HandleException
                 }
                 scheduleStore.put(finalSchedule);
                 final ScheduleRecordBuilder scheduleRecords = context.recordBuilder(ScheduleRecordBuilder.class);
-                scheduleRecords.scheduleID(finalSchedule.scheduleId());
-                scheduleRecords.scheduledTransactionID(HandlerUtility.transactionIdForScheduled(finalSchedule));
+                scheduleRecords
+                        .scheduleID(finalSchedule.scheduleId())
+                        .scheduledTransactionID(HandlerUtility.transactionIdForScheduled(finalSchedule));
```

Comment:
This reads better the way it was (also takes one less line).

Reply:
This change was the result of a merge.

Reply:
I noticed.  Comment is that the wrong change from the conflict was taken.
Easy enough to fix in the next PR if needed, however.